wp-content/themes/missilethreat/assets/_scss/abstracts/_mixins.scss

```diff
@@ -48,6 +48,16 @@
   font-size: rem($size);
 }
 
+//
```

Comment:
Jonathan, can I ask that you add a description of what this mixin does as a comment? Including a definition of what the parameters are & the format they need to be? Future us will be grateful. 🙏